cli/tools/upgrade.rs

```diff
@@ -15,12 +16,84 @@ use std::io::Write;
 use std::path::Path;
 use std::path::PathBuf;
 use std::process::Command;
+use std::time::Duration;
 
 static ARCHIVE_NAME: Lazy<String> =
   Lazy::new(|| format!("deno-{}.zip", env!("TARGET")));
 
 const RELEASE_URL: &str = "https://github.com/denoland/deno/releases";
 
+const UPGRADE_CHECK_INTERVAL: i64 = 12;
```

Comment:
The units would be more clear here if this was a Duration object.

12 seems very frequent. Suggest 24.

Reply:
`chrono`'s `Duration` object is not marked as `const` and can't be used here.